server/src/routers/userRouter.js

```diff
@@ -1,21 +1,29 @@
-const express = require('express')
-const path = require('path')
+const express = require("express");
+const path = require("path");
 const router = new express.Router()
 
-const User = require('../models/users')
+const User = require("../models/users")
 
-router.post('/signup', async (req, res) => {
-    console.log('Creating a new account for a new user')
+router.post("/user/signup", async (req, res) => {
     const user = new User(req.body)
-    console.log(req.body)
     try {
         await user.save()
         res.send(user)
-        console.log('The user has been created')
+        console.log("The user has been created")
     } catch (e) {
         res.status(400).send(e)
         console.log(e)
     }
-})
+});
 
-module.exports = router
\ No newline at end of file
+router.post("/user/login", async (req, res) => {
+    try {
+        console.log(req.body)
+        const user = await User.checkCredentials(req.body.email, req.body.password)
+        res.send(user)
+    } catch (e) {
+        res.status(400).send(e)
```

Comment:
Since we aim to make it a backend API, we should stick on sending json responses even for error conditions. I had say, send back {error: e}

Reply:
Sure!